File: Pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float change;
    int coin_number = 0;
    do
    {
        change = get_float("Change owed: ");
        change = round(change * 100);
    }
    while (change > 0 ? false : printf("Change owed: foo\n"));
    while (change > 0)
    {
        if (change >= 25)
        {
            change -= 25;
        }
        else if (change >= 10)
        {
            change -= 10;
        }
        else if (change >= 5)
        {
            change -= 5;
        }
        else
        {
            change --;
        }
        coin_number++;
    }
    printf("%i\n", coin_number);
}

File: Pset6/cash py/cash.py
from cs50 import get_float

while True:
    cash = get_float("cash owed: ")
    if cash > 0:
        cash = round(cash * 100)
        coins = 0
        while cash > 0:
            if cash / 25 >= 1:
                cash -= 25
            elif cash / 10 >= 1:
                cash -= 10
            elif cash / 5 >= 1:
                cash -= 5
            else:
                cash -= 1
            coins += 1
        print(coins)
        break
